tests/MongoDB.Driver.Tests/Specifications/auth/TestRunner.cs

```diff
@@ -0,0 +1,188 @@
+/* Copyright 2018-present MongoDB Inc.
```

Comment:
The new test runner to handle the auth tests, with the a subset of the tests ignored for now (ticket has been created to handle that subset at a later time).